MAVProxy/tools/MAVExplorer.py

```diff
@@ -428,9 +429,13 @@ def cmd_param(args):
 def cmd_loadfile(args):
     '''callback from menu to load a log file'''
     if len(args) != 1:
-        print "Error loading file"
+        fileargs = " ".join(args)
```

Comment:
cmd_loadfile should only expect 1 argument anyway - the name of the logfile to be loaded. Not sure why multiple arguments should be expected here.